requirements/edx/github.txt

```diff
@@ -21,6 +21,8 @@
 git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a0c695#egg=django-cas
 -e git+https://github.com/dgrtwo/ParsePy.git@7949b9f754d1445eff8e8f20d0e967b9a6420639#egg=parse_rest
 git+https://github.com/mfogel/django-settings-context-processor.git@b758c3930862761216267715a70bbefd206ac03a#egg=django-settings-context-processor==0.2
+git+https://github.com/SmileyChris/pyuca.git@555292b39692e2c8a13fbba02a284fb89c9940e4#egg=pyuca==1.0
```

Comment:
@muzaffaryousaf The changes have been merged into upstream and they have been released to PyPI, please move this requirement to base.txt and use the version on PyPI(https://pypi.python.org/pypi/pyuca/1.1)